pkg/epinio/components/application/AppProgressUpdateSource.vue

```diff
@@ -0,0 +1,261 @@
+<script lang="ts">
```

Comment:
There's huge C&P between this and `AppProgress`. The mode (create / edit) can be passed to `AppProgress` instead and the steps can be tweaked given that param